.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,100 @@
+name: Docker Image Build
+
+env:
+  IMAGE_NAME: pegasystems/pega-ready
+  VERSION: 2.1.0-${GITHUB_REPOSITORY}-${GITHUB_SHA}
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron:  '0 12 * * *'
+
+jobs:
+  docker-login:
+    name: Login to DockerHub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+  test:
+    name: Container Tests
+    runs-on: ubuntu-latest
+    if: always()
+    needs: docker-login
+    steps:
+    - name: Check out working repository
+      uses: actions/checkout@v2
+
+    - name: Run Container Tests
+      run: |
+        curl -LO https://storage.googleapis.com/container-structure-test/latest/container-structure-test-linux-amd64
+        chmod +x container-structure-test-linux-amd64
+        sudo mv container-structure-test-linux-amd64 /usr/local/bin/container-structure-test
+        make test
+        
+  scan:
+    name: Scan for CVEs
+    runs-on: ubuntu-latest
+    if: always()
+    needs: docker-login
+    steps:
+    - name: Check out working repository
+      uses: actions/checkout@v2
+      
+    - name: Build Image
+      run: make image
+
+    - name: Scan image
+      id: scan
+      uses: anchore/scan-action@v3
+      with:
+        image: "pegasystems/pega-ready:latest"
+        fail-build: true
+        severity-cutoff: "high"
+        acs-report-enable: true
+        
+    - name: Log Scan Results
+      if: always()
+      run: |
+        echo "Failures:"
+        cat ${{ steps.scan.outputs.sarif }} | jq '.runs[0].tool.driver.rules[].shortDescription.text' | grep -i "critical" || true
+        cat ${{ steps.scan.outputs.sarif }} | jq '.runs[0].tool.driver.rules[].shortDescription.text' | grep -i "high" || true
+        echo "Warnings:"
+        cat ${{ steps.scan.outputs.sarif }} | jq '.runs[0].tool.driver.rules[].shortDescription.text' | grep -i "medium" || true
```

Comment:
We should fail on medium so we can queue work to remediate and explicitly add them to grype